Trell/Protos/Work.proto

```diff
@@ -73,7 +73,7 @@ message Scheduled {
   google.protobuf.Timestamp timestamp = 2;
 }
 
-message Webhook {
+message Fetch {
```

Comment:
How about calling this type `Request`, even if the fieldname is fetch?  I'm thinking of the others, such as Upload, as nouns, not verbs.

Reply:
Yeah, I like that. Could even make Dynamic take a Request instead of a Json. (Later.)